cli/link.go

```diff
@@ -84,17 +86,43 @@ func (p PipelineLink) IsLocal() bool {
 	return p.FsAllow
 }
 
-//checks if the pipeline is up
-//otherwise it brings it up and fills the
-//link object
+// checks if the pipeline is up
+// otherwise it brings it up and fills the
+// link object
 func bringUp(pLink *PipelineLink) error {
 	alive, err := pLink.pipeline.Alive()
 	if err != nil {
-		if pLink.config[STARTING].(bool) {
-			alive, err = NewPipelineLauncher(pLink.pipeline,
-				pLink.config.ExecPath(), pLink.config[TIMEOUT].(int)).Launch(os.Stdout)
```

Comment:
> timeout has been removed
>
> It's usage in code is replaced by the previous default value 25

If we are keeping the support for launch scripts, it might make sense to also keep the timeout setting.